### ISSUE TITLE
KEEP_REPORTDIR does not work as expected

### ISSUE THREAD

**Issue by llxia** [AdoptOpenJDK/TKG#156]
When KEEP_REPORTDIR is set to false, we should not capture any passed test artifacts. However, this does not work as expected. Please see the build link below:
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.functional_s390x_linux_aot_Personal_testList_0/4/

It looks like all test artifacts are captured.

I think this is due to test pipeline uses boolean https://github.com/AdoptOpenJDK/openjdk-tests/blob/e55242930ee3bd56ceb2cbdcb898df130fbfc9c9/buildenv/jenkins/JenkinsfileBase#L78

and TKG uses string https://github.com/AdoptOpenJDK/TKG/blob/b7d19e42ab2c131d71ba037304b6efd8e07d5ad5/settings.mk#L195

**Comment by llxia** [comment]
We think we should change this line first
https://github.com/AdoptOpenJDK/openjdk-tests/blob/e55242930ee3bd56ceb2cbdcb898df130fbfc9c9/buildenv/jenkins/JenkinsfileBase#L78

**Comment by llxia** [comment]
@OscarQQ Could you help with this issue? Thanks

**Comment by smlambert** [comment]
Given https://github.com/AdoptOpenJDK/openjdk-tests/pull/2348 is now merged, can this be closed?

**Comment by llxia** [comment]
Not really. There are missing changes in https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/buildenv/jenkins/testJobTemplate#L37 and build scripts. For example:
https://github.com/eclipse/openj9/blob/bb13191b0bebc16d69eebfb72293e904858375da/buildenv/jenkins/common/variables-functions.groovy#L783
https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/a57a15186003e0990d94ceb06102ff2100666be3/pipelines/build/common/openjdk_build_pipeline.groovy#L292

We should not mix string and boolean value.

**Comment by smlambert** [comment]
Indeed.  I was looking through to tag good first issues, ones where there is a clear step by step description of the changes required.  I will leave this open, but not tagged as 'good first issue' for now.
